### PR TITLE
Update SpendAvailable in ExecuteTransaction

### PR DESCRIPTION
**Pull request by i-stam**
addresses #425

### CODE REVIEW

**Comment by i-stam** [comment]
we can possibly rename _topUpLimitDay and spendLimitDay, since they are merely timestamps.

File: contracts/wallet.sol [hunk 1]
```diff
@@ -294,9 +294,8 @@ contract SpendLimit is Controllable, Ownable {
     /// @dev Update available spend limit based on the daily reset.
     function _updateSpendAvailable() internal {
         if (now > _spendLimitDay.add(24 hours)) {
-            // Advance the current day by how many days have passed.
-            uint extraDays = now.sub(_spendLimitDay).div(24 hours);
-            _spendLimitDay = _spendLimitDay.add(extraDays.mul(24 hours));
+            // Advance the current timestamp.
+            _spendLimitDay = now;
```
Comment (mischat) [inline contracts/wallet.sol]: This looks sane, thank you ... I guess we should port this, and the tests we now have added to the 2.X branch. @i-stam

File: contracts/wallet.sol [hunk 2]
```diff
@@ -453,6 +452,7 @@ contract Wallet is Vault {
 
     using SafeMath for uint256;
 
+    string constant public WALLET_VERSION = "1.5.8";
```
Comment (mischat) [inline contracts/wallet.sol]: I would put the v1.5.8, as that is what we are doing in the repo. The tag is "v1.5.8"

File: contracts/wallet.sol (hunk 1 above)
Comment (i-stam) [reply contracts/wallet.sol]: I 'll do it

**Pull request merged**